opensrp-app/src/main/java/org/smartregister/SyncConfiguration.java

```diff
@@ -31,4 +31,7 @@ public boolean disableSyncToServerIfUserIsDisabled() {
         return false;
     }
 
+    public abstract SyncFilter getEncryptionParam();
```

Comment:
Can't this method be implemented with a default value so that only implementations that want override have to implement

Reply:
Same for this, we want it to be a choice at the application level